lib/Smokeping/sorters/Loss.pm

```diff
@@ -18,7 +18,7 @@ Call the sorter in the charts section of the config file
 
  ++ loss
   sorter = Loss(entries=>10)
-  title = The Loosers
+  title = The Losers
```

Comment:
Looser is a word, but it's roughly the result of loosening something.
Loser is also a word, and it's what happens when something loses.